src/dsc/Microsoft.PowerToys.Configure/examples/configureLauncherPlugins.dsc.yaml

```diff
@@ -0,0 +1,17 @@
+properties:
+  resources:
+    - resource: PowerToysConfigure
+      directives:
+        description: Configure PowerToys
+      settings:
+        Debug: true
+        PowerLauncher:
+          Enabled: true
+          AdditionalProperties:
```

Comment:
Plugins have multiple things that do not comply with the rest of the settings:
- They're stored separately from `Properties` field unlike all other settings, see `PowerLauncherSettings`
- They are stored in a `IEnumerable` which makes identifying them difficult, thus the inferring of "key" property (`Name` only at the moment) in `SetAdditionalSettingsCommandLineCommand.Execute`. Also, it's not clear whether we should append/combine/overwrite the elements in `IEnumerable`.

So yeah, currently they exist as a kind of special case due to their structure and extensibility.

Reply:
@yuyoyuppe
For policies we use the plugin id as identification because this is the only thing that is 100% unique. From what I have in mind PowerLauncher deduplicates based on the id and newest version/date.

Reply:
Why is it named "AdditionalProperties", though? Is that the name in the json file?

Reply:
So ti means we can just have "Plugins" instead of "Additional Properties>Plugins"? That would be preferred.

Reply:
The way it works is more general than that though 🤔. Currently, it's a bit limited in terms of the supported types, but the logic could be extended to support all the properties, i.e. not only Plugins for PowerRun, but for the rest of the properties as well.